lib/inferno/app/endpoint.rb

```diff
@@ -1,16 +1,20 @@
 # frozen_string_literal: true
 
 require 'sinatra/base'
+require 'sinatra/config_file'
 require 'sinatra/custom_logger'
 require 'sinatra/cookies'
 require_relative 'helpers/configuration'
 require_relative 'helpers/browser_logic'
+require_relative 'models/module'
+
 module Inferno
   class App
     class Endpoint < Sinatra::Base
+
       register Sinatra::ConfigFile
 
-      config_file File.join('..', '..', ENV['INFERNO_CONFIG_FILE'] || 'config.yml')
+      config_file File.join('..', '..', '..', ENV['INFERNO_CONFIG_FILE'] || 'config.yml')
```

Comment:
I think we'll need a new methodology for this if Inferno is used as a gem.

Reply:
Agreed.  I can work on that if it belongs in this PR.  I was trying to minimize the amount of changes as much as I could because a lot more things (like `requires`, the erb templates, and the testing instance models) also have to change, but it would be a huge overhaul at once.

I'll look into improving this for now though!